package.json

```diff
@@ -73,7 +73,7 @@
 		"build-desktop:source": "run-s build-desktop:config build-desktop:server build-desktop:client build-desktop:static",
 		"build-desktop:config": "node desktop/bin/build-desktop-config.js",
 		"build-desktop:secrets": "node desktop/bin/build-desktop-secrets.js",
-		"build-desktop:server": "BROWSERSLIST_ENV=server CALYPSO_ENV=desktop webpack --config client/webpack.config.desktop.js --stats-preset errors-only",
+		"build-desktop:server": "BROWSERSLIST_ENV=server CALYPSO_ENV=desktop webpack --config client/webpack.config.desktop.js --stats-preset errors-only $WEBPACK_OPTIONS",
```

Comment:
I don't see this one being called by any other task that sets `WEBPACK_OPTIONS`, is it really needed here?

Reply:
Maybe not. My reasoning for this change was:

- Add `$WEBPACK_OPTIONS` to any command that calls `webpack` to avoid future surprises
- Replace usages of `PROGRESS=true` with `WEBPACK_OPTIONS='--progress=profile'`, as they serve the same purpose. That currently means the Docker file (https://github.com/Automattic/wp-calypso/pull/51269/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R23) and the script `build-client-stats` (https://github.com/Automattic/wp-calypso/pull/51269/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R97).

If we want to refine it, we can probably remove `WEBPACK_OPTIONS='--progress=profile'` from `build-client-stats` and set it up in ICFY env vars (as far as I know only ICFY is using that script on a regular basis), and set it up in the Desktop build as well